tests/playwright-test/match-grep.spec.ts

```diff
@@ -87,3 +87,15 @@ test('should grep invert test name', async ({ runInlineTest }) => {
   expect(result.skipped).toBe(0);
   expect(result.exitCode).toBe(0);
 });
+
+test('should be case insensitive by default', async ({ runInlineTest }) => {
+  const result = await runInlineTest(files, { 'grep': 'TesT Cc' });
```

Comment:
I wonder if it actually a regex or just a substring in this case?

Reply:
The first test case in this file implies that it is a regex.